### ISSUE TITLE
Feature Request: Resource for managing AWS Glue Developer Endpoints

### ISSUE THREAD

**Issue by bflad** [hashicorp/terraform-provider-aws#3876]
Split feature request from #1416. For issues not immediately being worked on, please use :+1: upvotes on this original issue comment to help guage community interest.

### Terraform Version

```
terraform 0.10+
terraform-provider-aws (new feature)
```

### Affected Resource(s)

- `aws_glue_dev_endpoint`

### Terraform Configuration Files

```hcl
# New Resource - not implemented
resource "aws_glue_dev_endpoint" "example" {
  # ...
}
```

### Expected Behavior

Create, read, update, and delete support for AWS Glue developer endpoints.

### Actual Behavior

Resource not implemented.

### References

* https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-dev-endpoint.html
* https://docs.aws.amazon.com/sdk-for-go/api/service/glue/#Glue.CreateDevEndpoint

**Comment by rangerscience** [comment]
Can this be re-opened?

I'm interested in trying to contribute to Terraform (love this product, you guys, it's amazing), this seems like a reasonable place to start... but I'm not that familiar with Go or the Terraform code (just it's usage). Any pointers? Both the raw "getting setup to dev TF" (although there's stuff in the contribution guidelines for that) and the "how to *effectively* stare at our code" would be appreciated.

My use case it making it easy to boot up (and tear down) a Zeppelin notebook; they're amazing, but expensive to keep running.

I have a working CloudFormation template for the dev endpoint (if that would be helpful); the Zeppelin CF template then relies on some values that come from the dev endpoint that are accessible via the AWS CLI; presumably, this means if there's a Terraform resource for the dev endpoint, they'd be usable in a CF template managed by TF, even if a Zeppelin notebook resource is beyond the TF scope (which... is likely, looking at the CF template).

**Comment by xmj** [comment]
Out of curiosity, has this feature request already been taken into the roadmap for the TF AWS provider?

**Comment by jckuester** [comment]
I added a PR that adds the resource `aws_glue_dev_endpoint`: https://github.com/terraform-providers/terraform-provider-aws/pull/7895

**Comment by xmj** [comment]
Thanks @jckuester!

**Comment by DrFaust92** [comment]
Closed via #7895
